packages/common/src/index.test.ts

```diff
@@ -214,4 +224,43 @@ describe('Common index', () => {
             expect(formatItemValue(tableCalculation, 5)).toEqual('5');
         });
     });
+
+    describe('default values on filter rule', () => {
+        // TODO mock some timezones
```

Comment:
Added some tests, I'd like to test with different timezones, but it is not easy to mock moment.. maybe in some other PR